### ISSUE TITLE
RmlUi and OpenSceneGraph

### ISSUE THREAD

**Issue by triblatron** [mikke89/RmlUi#117]
Hi,

I have resurrected the integration of OpenSceneGraph with RmlUi from when it was called osgRocket.  Now called osgRmlUi
Unfortunately, despite successfully loading the window.rml template and invader.rcss stylesheet, form controls are not being rendered correctly.  There is text but no surrounding control background.  Is this something you have seen before?

**Comment by mikke89** [comment]
Hi!

There shouldn't be anything special about form controls. I suspect that the `invader.tga` asset used in invader.rcss is not properly loaded, or located. Can you check the log output from RmlUi?

**Comment by triblatron** [comment]
Hi,

Thanks for your reply.

There are no warnings about failing to load invader.tga.  The only messages are related to loading fonts - all successful.  The window background and title bar appear correctly.  

It turns out that I wasn't calling Rml::Controls::Initialise() or linking with RmlControls.  I found it by comparing to invaders/src/main.cpp.  Should it be in the initialisation section of the manual or is it one of the things that will be simplified in the upcoming release?

**Comment by mikke89** [comment]
Ah, I see. You're right, this behavior was recently changed. There is no longer a need to initialize it. The online documentation reflects the current state of the master branch. This could be clearer in the documentation, so I added a big note there. Also, I released [offline documentation for RmlUi 3.3](https://github.com/mikke89/RmlUiDoc/releases/tag/3.3) which may be helpful.

I don't expect to normally do releases on the documentation, but hope it can be helpful in this case with the breaking changes.
